.travis.yml

```diff
@@ -137,3 +139,11 @@ jobs:
       #language: ruby  # osx does not support language=python
       #env: CONDA_TOKEN=$CONDA_UPLOAD_TOKEN_TAG
 
+    - stage: gen_release
+      name: "Generate Release"
+      install:
+         - git clone https://github.com/slaclab/ruckus.git -b new_release
```

Comment:
Will this eventually need to be the master branch instead of new_release?

Reply:
Yes. That will need to be updated once Ruckus is released.